test/helper-review-pull-request.ts

```diff
@@ -107,6 +112,70 @@ describe('reviewPullRequest', () => {
     expect(numMockedHelpersCalled).equals(4);
   });
 
+  it('Succeeds when diff string provided', async () => {
+    const diffString = readFileSync(
+      resolve(fixturePath, 'many-to-many.diff')
+    ).toString();
+    let numMockedHelpersCalled = 0;
+    const validFileHunks = new Map<string, Hunk[]>();
+    validFileHunks.set('cloudbuild.yaml', [
+      {
+        newStart: 0,
+        newEnd: 10,
+        oldStart: 0,
+        oldEnd: 10,
+        newContent: [],
+      },
+    ]);
+
+    const stubMakePr = proxyquire.noCallThru()(
```

Comment:
I feel like we probably should be able to use sinon, rather than proxyquire here? It feels a bit weird to me to have both approaches to mocking. I've been landing on a pattern where, if I'm tempted to use `proxyquire`, I'll instead encapsulate behavior inside a helper method and mock that with sinon.